### PR TITLE
I have added script for basic number guessing game. 

### PR DESCRIPTION
**Pull request by ultraUnicorn74**
please review this.

### CODE REVIEW

**Review by Gaurav451 — CHANGES_REQUESTED** [review]
In line 4,we can stop the restriction from 1 to 99 by asking the end point from the user,say it be a.Then line 4 will become randint(1,a).

**Comment by ultraUnicorn74** [comment]
@Gaurav451  Actually here i am asking the user to enter any random number between 1 to 99 only so that the user have to guess number between 1 to 99 only . please suggest if my approach is not right.

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
@richabh456 
You can ask the user the limit & chances for guessing.
It will make script more dynamic

**Review by ultraUnicorn74 — COMMENTED** [review]
@SKAUL05 Sir i have made the required changes in code, Please review this.

File: Basic-Scripts/guessinggame.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    win = False
+    while chances > 0:
+        guess = int(input())
+        if guess == number:
+            print("HOORAY YOU WON!!!")
+            win = True
+            break
+        elif guess < number:
+            print("Your guess was too low: Guess a number higher than", guess)
+            win = False
```
Comment (SKAUL05) [inline Basic-Scripts/guessinggame.py]: No need of this

File: Basic-Scripts/guessinggame.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        else:
+            print("Your guess was too high: Guess a number lower than", guess)
+            win = False
```
Comment (SKAUL05) [inline Basic-Scripts/guessinggame.py]: No need of this

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
Everything else LGTM

**Comment by ultraUnicorn74** [comment]
@SKAUL05  sir,now is it fine?

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
Only win = False was required to remove not everything else
U need to print whether number is lower or greater

**Comment by ultraUnicorn74** [comment]
@SKAUL05 yes sir got it, Thank you !is it okay now?

File: Basic-Scripts/guessinggame.py (hunk 2 above)
Comment (SKAUL05) [reply Basic-Scripts/guessinggame.py]: Only remove win= False from this and above else remains same

**Comment by SKAUL05** [comment]
> @SKAUL05 yes sir got it, Thank you !is it okay now?

Pls have a look at the comment above

**Comment by ultraUnicorn74** [comment]
@SKAUL05 Please review this.

**Review by SKAUL05 — APPROVED** [review]
Good to go :)

**Pull request merged**